### PR TITLE
Removes RubyConf PT banner

### PR DESCRIPTION
**Pull request by joaojusto**

### CODE REVIEW

**Comment by gabrielpoca** [comment]
CI failing

**Comment by joaojusto** [comment]
I don't have access to semaphore to check the error, on my local are only Rubocop offenses that are already on the master branch

**Comment by naps62** [comment]
@joaojusto master is green on semaphore, so something's not right.
and that's probably due the fact that you're updating the Gemfile.lock. why is that?
rubocop also seems to be updated to 0.34, which is probably why your local master fails as well

File: Gemfile.lock [hunk 1]
```diff
@@ -353,16 +356,16 @@ GEM
...
+    tilt (2.0.1)
     truncate_html (0.9.3)
     turbolinks (2.5.3)
       coffee-rails
     tzinfo (1.2.2)
       thread_safe (~> 0.1)
-    uglifier (2.7.1)
+    uglifier (2.7.2)
       execjs (>= 0.3.0)
       json (>= 1.8.0)
```
Comment (naps62) [inline Gemfile.lock]: why did you update this? this should be removed to it's own PR if it was really intended

**Comment by joaojusto** [comment]
@naps62 my bad, I had an error on my dependencies and updated the gems. Now everything should be working

**Comment by gabrielpoca** [comment]
:+1:
